crates/accelerate/README.md

```diff
@@ -8,3 +8,27 @@ place to put the code.  This is _usually_ the right place to put new Rust/Python
 The `qiskit-pyext` crate is what actually builds the C extension modules.  Modules in here should define
 themselves has being submodules of `qiskit._accelerate`, and then the `qiskit-pyext` crate should bind them
 into its `fn _accelerate` when it's making the C extension.
+### Adding a module to `_accelerate`
+Specifically, adding a rust module `my_module` to Qiskit such that it can be called from within Python involves
+the following steps:
+1. Add a module function to your to your `my_module.rs` file, it should look similar to this:
+  
+     ```rust
+     pub fn my_module(m: &Bound<PyModule>) -> PyResult<()> {}
+     ```
```

Comment:
These actually usually _ought_ to be tagged with a `#[pymodule]` attribute, but since #12889, this is actually temporarily correct.  It'll change to being `#[pymodule(submodule)]` when we update to PyO3 0.22, though.